tests/integration/ceph_base_file_test.go

```diff
@@ -171,11 +171,13 @@ metadata:
 spec:
   containers:
   - name: ` + podName + `
-    image: busybox
+    image: krallin/ubuntu-tini
```

Comment:
nice, hopefully this helps stabilize the file tests!

Reply:
It makes it so the pod can actually be killed by k8s when `kubectl delete` is issued, which is great. I really struggled with this in my local env.